### PR TITLE
Update admonition styles - Closes #1359

### LINKED ISSUE #1359: Update Admonitions style
## Which content is outdated?
Style for Admonitions:

- Info
- Tip
- Important
- Warning

## What should be the new content?
image

Epic: #1294

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #1359

### CODE REVIEW

**Comment by shuse2** [comment]
@Tschakki 
I think the size (width/ height) is slightly off and letter is bit smaller

File: ui/src/css/extensions/common-custom.css [hunk 1]
```diff
@@ -450,3 +458,80 @@ footer.footer {
...
+.doc .admonitionblock.warning td.content {
+  background: rgba(var(--rgb-fury-red), 0.07);
+}
+
+.doc .admonitionblock .icon {
+  transform: unset;
+  border-bottom-left-radius: unset;
+  border-top-right-radius: unset;
+  height: 2rem;
+  padding: 0 1.5rem;
```
Comment (shuse2) [inline ui/src/css/extensions/common-custom.css]:
```suggestion
  height: 2.3rem;
  padding: 0 1rem;
```

**Review by shuse2 — COMMENTED** [review]
could you try this value?

**Comment by Tschakki** [comment]
new updates are deployed 👍

**Review by shuse2 — APPROVED** [review]

**Pull request merged**